Gemfile.lock

```diff
@@ -126,4 +157,4 @@ DEPENDENCIES
   valid_url!
 
 BUNDLED WITH
-   1.14.6
+   1.17.3
```

Comment:
Is this bundlers version? I'm not sure what line 159 and 160 mean.

Reply:
Yes @farrahh! bundler is the gem that provides the command `bundle` as in `bundle install`. https://bundler.io/ 

This line says what version of bundler was used to install the gems and generate this Gemfile.lock